lib/super_digit.rb

```diff
@@ -1,11 +1,14 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - o(n)
+# Space Complexity - o(n)
 def super_digit(n)
```

Comment:
This works but the time complexity is O(n^2) where n is the number of digits because the map and sum operations are O(n)